Update homework.md

DavoodKhoshnood opened the pull request:
I've updated and fixed some answers and one question of homework.md

<!--

The title for your pull request should be made in this format

CITY CLASS_NO - FIRST_NAME LAST_NAME - MODULE - WEEK_NO

For example,

London Class 7 - Chris Owen - HTMl/CSS - Week 1

-->

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

SamanZahedi commented:
Well done Davood jan, very neat and right to the point 
Just add Git documentation to your cheating stuff :D

DavoodKhoshnood commented:
Update homework

DavoodKhoshnood commented:
> I've updated and fixed some answers and one question of homework.md
> 
> 
> 
> <!--
> 
> 
> 
> The title for your pull request should be made in this format
> 
> 
> 
> CITY CLASS_NO - FIRST_NAME LAST_NAME - MODULE - WEEK_NO
> 
> 
> 
> For example,
> 
> 
> 
> London Class 7 - Chris Owen - HTMl/CSS - Week 1
> 
> 
> 
> -->
> 
> 
> 
> # Your Details
> 
> 
> 
> - Your Name: Davood Khoshnood
> 
> - Your City: London
> 
> - Your Slack Name:Davood
> 
> 
> 
> # Homework Details
> 
> 
> 
> - Module: Git & GitHub
> 
> - Week: 1
> 
> 
> 
> -----
> [View rendered homework.md](https://github.com/DavoodKhoshnood/GitHomeworkFixErrors/blob/main/homework.md)

AliNurisiahsiahi commented:
Great Davood
